src/components/Form/Form.tsx

```diff
@@ -0,0 +1,169 @@
+import { FieldArray, Formik, Field } from 'formik';
+import * as Yup from 'yup';
+import {
+  Root,
+  Title,
+  AddForm,
+  InputLabel,
+  TextFieldInput,
+  LargeTextFieldInput,
+  SelectInput,
+  TimesContainer,
+  RemoveTimeButton,
+  AddTimeButton,
+  ErrorMessage
+} from './Form.styles';
+import { v4 as uuidv4 } from 'uuid';
+import axios from 'axios';
+
+const validationSchema = Yup.object({
+  name: Yup.string().required().min(3),
+  description: Yup.string(),
+  image: Yup.string(),
+  times: Yup.array().of(
+    Yup.string()
+  ).min(1).required(),
+  notification: Yup.string().required().oneOf(
+    ['email', 'phone', 'text']
+  )
+});
+
+interface Medication {
+  name: string,
+  description: string,
+  image: string,
+  times: string[],
+  notification: string
+}
```

Comment:
Not priority, but I also created a Medication interface - once this is merged, we can export/import one of them